rpc/common/service.proto

```diff
@@ -38,13 +38,14 @@ message Package {
   string version = 2;
   string release = 3;
   int32  epoch   = 4;
-  string arch    = 5;
+  PkgIdentifier identifier = 5;
+  string arch    = 6;
```

Comment:
Previous numbers should not be changed - https://protobuf.dev/programming-guides/proto3/#assigning
```suggestion
  PkgIdentifier identifier = 19;
  string arch    = 5;
```

Reply:
done, see https://github.com/aquasecurity/trivy/pull/5439/commits/37a098c5b5d3c1396eb6d830cf70588b8b4b219c